crates/bitwarden-threading/README.md

```diff
@@ -0,0 +1,11 @@
+# bitwarden-threading
+
+Utility crate for Bitwarden SDK to handle threading and async quirks in FFI contexts.
+
+## WASM Testing
+
+To run the WASM tests, you can use the following command:
+
+```bash
+cargo test --target wasm32-unknown-unknown --all-features -- --nocapture
```

Comment:
Should we add this to the CI test as well?

Reply:
We probably should but I'm getting too many compilation errors to fix in this PR 🥲

Reply:
We can probably just test this crate only:
```
cargo test --target wasm32-unknown-unknown -p bitwarden-threading --all-features -- --nocapture
```